### PR TITLE
Rails-React

### PR DESCRIPTION
**Pull request by degisew**
## Rails React
**In this PR, I implemented the following features:**
 - Created a rails API app using `rails new hello-rails-back-end --api`
 - Created a separate React project using `npx create-react-app`
 -  Used Redux to manage state globally
 -  Used the rails endpoints on my react UI and displayed it.

### CODE REVIEW

**Comment by DeliceLydia** [comment]
## Hi @degisew, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have an open PR on the frontend, please create it from a feature to the dev branch and write a README

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by degisew** [comment]
Okay, Thanks

File: README.md [hunk 1]
```diff
@@ -1,24 +1,218 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/degisew/hello-rails-backend/blob/dev/LICENSE) licensed.
+
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] Kindly fix this link is broken and it's giving me a 404 error. This way the readme file will look professional without broken links.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @degisew, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
👍 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add 5 greetings (messages) to the app. You can do this using the seed file in the db folder. This is a requirement, and the only way to add is by using the seed file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @degisew    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**